test/test_functional_tensor.py

```diff
@@ -144,6 +144,12 @@ def test_rotate_batch(self, device, dt):
         center = (20, 22)
         _test_fn_on_batch(batch_tensors, F.rotate, angle=32, interpolation=NEAREST, expand=True, center=center)
 
+    def test_rotate_interpolation_type(self):
+        tensor, _ = _create_data(26, 26)
+        res1 = F.rotate(tensor, 45, interpolation=2)
```

Comment:
should we use `PIL.Image.BILINEAR` instead?
(here and in other places)

Reply:
I was just putting back the original content, but happy to change to the named constants if that is ok with you.